packages/Sandblocks-Smalltalk/SBStMessageSend.class.st

```diff
@@ -446,7 +446,7 @@ SBStMessageSend >> printOn: aStream [
 { #category : #'as yet unclassified' }
 SBStMessageSend >> receiver [
 
-	(self owner notNil and: [self isInCascade]) ifTrue: [self owner receiver].
+	(self owner notNil and: [self isInCascade]) ifTrue: [^ self owner receiver].
```

Comment:
@tom95 This breaks two cascade tests in `SBASTNodeTest`, could you take a look and let me know whether these tests are correct or not? :)

Reply:
Fixed via https://github.com/tom95/sandblocks/pull/86/commits/ba58815f1e26d7cf7e3ee24d61bab3090ab893c5